requirements-dev.txt

```diff
@@ -1 +1,2 @@
 pytest==6.2.5
+python-dotenv
```

Comment:
@PCBush 
after activating your deployment \ environment (`.venv\Scripts\activate.bat`) do

`pip install -r requirements-dev.txt to get `